src/lib/modules/blockchain_process/httpProxyOverride.js

```diff
@@ -0,0 +1,123 @@
+/* global require */
+
+const http = require('http');
+const https = require('https');
+const httpProxyWsIncoming = require('http-proxy/lib/http-proxy/passes/ws-incoming');
+const common = require('http-proxy/lib/http-proxy/common');
```

Comment:
Since this is a newly introduced file, it'd be great if we could go with `import` statements right away